google/cloud/storage/_opentelemetry_meter.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2014 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Manages OpenTelemetry metrics creation and handling"""
+
+try:
+    from opentelemetry import metrics
+    from opentelemetry.sdk.metrics import Counter, MeterProvider
+    from opentelemetry.exporter.cloud_monitoring import (
+        CloudMonitoringMetricsExporter,
+    )
+
+    HAS_OPENTELEMETRY_INSTALLED = True
+except ImportError:
+    HAS_OPENTELEMETRY_INSTALLED = False
+
+FUNCTION_NAME_KEY = 'instrumented_function_name'
+
+if HAS_OPENTELEMETRY_INSTALLED:
+    meter = metrics.get_meter(__name__)
+    metrics.get_meter_provider().start_pipeline(
+        meter, CloudMonitoringMetricsExporter(), 15
```

Comment:
What is this 15? Can you use a named parameter here?

Reply:
Made it named. This number denotes how frequently the Exporter thread will export things (once every X seconds).